Code/Editor/PluginManager.cpp

```diff
@@ -18,8 +18,8 @@
 #include "Include/IPlugin.h"
 
 
-typedef IPlugin* (* TPfnCreatePluginInstance)(PLUGIN_INIT_PARAM* pInitParam);
-typedef void (* TPfnQueryPluginSettings)(SPluginSettings&);
+using TPfnCreatePluginInstance = IPlugin *(*)(PLUGIN_INIT_PARAM *pInitParam);
+using TPfnQueryPluginSettings = void (*)(SPluginSettings &);
```

Comment:
nit: If you select these lines and run clang-format from VS the `*` and `&` tokens will be moved to the right place (ideally should be left aligned)